### PR TITLE
Add a channel subcommand to add all visible channel

### LINKED ISSUE #83: Allow all channel to be added at once
Allow `all` as channel name.
This can not be done in the addchannel command and will therefore require another subcommand to support this as a slash command.

- [x] Add slash command for `addAllChannel`
- [x] Add text command for `addAllChannel`
- [x] Add Button to add all channel to the setup command.

### PR DESCRIPTION
**Pull request by RainbowDashLabs**
Add a subcommand to add all channel as reputation channel.
Add a button in setup command to do the same.

Close #83

### CODE REVIEW

File: src/main/java/de/chojo/repbot/commands/Channel.java [hunk 1]
```diff
@@ -118,13 +126,31 @@ private boolean add(MessageEventWrapper eventWrapper, CommandContext context) {
...
+    }
+
+    private boolean addAll(MessageEventWrapper eventWrapper) {
+        addAllChannel(eventWrapper.getGuild());
+        eventWrapper.reply(loc.localize("command.channel.sub.addAll.added", eventWrapper.getGuild())).queue();
+        return true;
+    }
+
+    private void addAllChannel(Guild guild) {
+        guild.getTextChannels().stream()
```
Comment (yannicklamprecht) [inline src/main/java/de/chojo/repbot/commands/Channel.java:141]: duplicated 1

File: src/main/java/de/chojo/repbot/commands/Setup.java [hunk 2]
```diff
@@ -204,7 +206,16 @@ private void buildChannelsButton(ButtonDialog buttons) {
             c.reply(localizer.localize("command.setup.dialog.setupComplete", c.getGuild()))
                     .queue();
             return Result.finish();
-        }));
+        })).add(Button.primary("all", "command.setup.dialog.channels.allChannel"), c ->{
+            var guild = c.getGuild();
+            var self = guild.getSelfMember();
+            guild.getTextChannels().stream()
```
Comment (yannicklamprecht) [inline src/main/java/de/chojo/repbot/commands/Setup.java:212]: duplicate ref 1

**Review by yannicklamprecht — CHANGES_REQUESTED** [review]

File: src/main/java/de/chojo/repbot/commands/Setup.java [hunk 3]
```diff
@@ -204,7 +206,16 @@ private void buildChannelsButton(ButtonDialog buttons) {
             c.reply(localizer.localize("command.setup.dialog.setupComplete", c.getGuild()))
                     .queue();
             return Result.finish();
-        }));
+        })).add(Button.primary("all", "command.setup.dialog.channels.allChannel"), c ->{
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/commands/Setup.java:209]:
```suggestion
        })).add(Button.primary("all", "command.setup.dialog.channels.allChannel"), c -> {
```

**Review by yannicklamprecht — APPROVED** [review]

File: src/main/java/de/chojo/repbot/commands/Channel.java [hunk 4]
```diff
@@ -118,13 +126,25 @@ private boolean add(MessageEventWrapper eventWrapper, CommandContext context) {
...
+        event.reply(loc.localize("command.channel.sub.addAll.added", event.getGuild())).queue();
+    }
+
+    private boolean addAll(MessageEventWrapper eventWrapper) {
+        FilterUtil.getAccessableTextChannel(eventWrapper.getGuild()).forEach(c -> guildData.addChannel(eventWrapper.getGuild(), c));
+        eventWrapper.reply(loc.localize("command.channel.sub.addAll.added", eventWrapper.getGuild())).queue();
+        return true;
+    }
+
+
```
Comment (Taucher2003) [inline src/main/java/de/chojo/repbot/commands/Channel.java:140]:
```suggestion
```

**Pull request merged**